[2025-02-14T00:37:18.425+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-14T00:37:18.460+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: third_dag.task3 manual__2025-02-14T00:37:09.922249+00:00 [queued]>
[2025-02-14T00:37:18.477+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: third_dag.task3 manual__2025-02-14T00:37:09.922249+00:00 [queued]>
[2025-02-14T00:37:18.478+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-14T00:37:18.496+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): task3> on 2025-02-14 00:37:09.922249+00:00
[2025-02-14T00:37:18.501+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=542) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-14T00:37:18.503+0000] {standard_task_runner.py:72} INFO - Started process 544 to run task
[2025-02-14T00:37:18.503+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'third_dag', 'task3', 'manual__2025-02-14T00:37:09.922249+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/03_third_dag.py', '--cfg-path', '/tmp/tmpjuzfeq6j']
[2025-02-14T00:37:18.505+0000] {standard_task_runner.py:105} INFO - Job 57: Subtask task3
[2025-02-14T00:37:18.559+0000] {task_command.py:467} INFO - Running <TaskInstance: third_dag.task3 manual__2025-02-14T00:37:09.922249+00:00 [running]> on host ab74d4d90d32
[2025-02-14T00:37:18.661+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='third_dag' AIRFLOW_CTX_TASK_ID='task3' AIRFLOW_CTX_EXECUTION_DATE='2025-02-14T00:37:09.922249+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-14T00:37:09.922249+00:00'
[2025-02-14T00:37:18.662+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-14T00:37:18.682+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-14T00:37:18.683+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'sleep 5']
[2025-02-14T00:37:18.694+0000] {subprocess.py:99} INFO - Output:
[2025-02-14T00:37:23.697+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-14T00:37:23.726+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-14T00:37:23.727+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=third_dag, task_id=task3, run_id=manual__2025-02-14T00:37:09.922249+00:00, execution_date=20250214T003709, start_date=20250214T003718, end_date=20250214T003723
[2025-02-14T00:37:23.824+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-14T00:37:23.848+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-14T00:37:23.850+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
